ci is failing

```
FAILED: src/nvim/CMakeFiles/nvim.dir/fileio.c.o 
/usr/bin/cc -DEXITFREE -DINCLUDE_GENERATED_DECLARATIONS -DMIN_LOG_LEVEL=3 -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -D_GNU_SOURCE -Dnvim_EXPORTS -Icmake.config -I../src -Isrc/nvim/auto -Iinclude -I../.deps/_usr/include/luajit-2.1 -isystem ../.deps/_usr/include -isystem /usr/local/include -DNVIM_TS_HAS_SET_MATCH_LIMIT -DNVIM_TS_HAS_SET_ALLOCATOR -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -Werror -MD -MT src/nvim/CMakeFiles/nvim.dir/fileio.c.o -MF src/nvim/CMakeFiles/nvim.dir/fileio.c.o.d -o src/nvim/CMakeFiles/nvim.dir/fileio.c.o -c ../src/nvim/fileio.c
../src/nvim/fileio.c:5302:43: error: expected ';' after expression
  memchrsub(user, '\\', '_', sizeof(user))

--- a/src/nvim/fileio.c
+++ b/src/nvim/fileio.c
@@ -5296,7 +5296,6 @@ static void vim_mktempdir(void)
   char user[40] = { 0 };
 
   (void)os_get_username(user, sizeof(user));
-
   // Usernames may contain slashes! #19240
   memchrsub(user, '/', '_', sizeof(user));
   memchrsub(user, '\\', '_', sizeof(user));

--- a/src/nvim/fileio.c
+++ b/src/nvim/fileio.c
@@ -5297,6 +5297,10 @@ static void vim_mktempdir(void)
 
   (void)os_get_username(user, sizeof(user));
 
+  // Usernames may contain slashes! #19240
+  memchrsub(user, '/', '_', sizeof(user));
+  memchrsub(user, '\\', '_', sizeof(user));
+
   // Make sure the umask doesn't remove the executable bit.
   // "repl" has been reported to use "0177".
   mode_t umask_save = umask(0077);

